tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/AspNetMvc5IastTests.cs

```diff
@@ -28,6 +28,22 @@ public AspNetMvc5IntegratedWithIast(IisFixture iisFixture, ITestOutputHelper out
         : base(iisFixture, output, classicMode: false, enableIast: true)
     {
     }
+
+    [SkippableTheory]
+    [Trait("Category", "ArmUnsupported")]
+    [Trait("RunOnWindows", "True")]
+    [Trait("LoadFromGAC", "True")]
+    [InlineData("text/html", 200, "nosniff")]
+    [InlineData("text/html", 200, "")]
+    [InlineData("application/xhtml%2Bxml", 200, "")]
+    [InlineData("text/plain", 200, "")]
+    [InlineData("text/html", 200, "dummyvalue")]
+    [InlineData("text/html", 500, "")]
```

Comment:
nit: you might also want to test a content-type that includes a `charset` too? No reason it should fail AFAICT, just for extra testing 🙂

Reply:
Sure. Added!